events/wol_war_events.txt

```diff
@@ -4214,9 +4214,9 @@ character_event = {
 	trigger = {
 		has_focus = focus_war
 		OR = {
-			trait = light_foot_leader
-			trait = heavy_infantry_leader
-			trait = cavalry_leader
+			trait = infantry_commander_leader
+			trait = infantry_commander_leader
```

Comment:
Now in the code there is a huge number of repetitions of the same conditions.

Reply:
Hmmmm, backtrack time

Reply:
Maybe heavy_infantry_leader should be replaced with something that gives a bonus to elite infantry (or what is this thing called)?